lingvo/lingvo.bzl

```diff
@@ -1,8 +1,7 @@
 """Implements custom rules for Lingvo."""
 
 def tf_copts():
-    # TODO(drpng): autoconf this.
-    return ["-std=c++17", "-Wno-sign-compare", "-mavx"] + select({
```

Comment:
This is handled in the `write_to_bazelrc` already!

Reply:
Hi, I think removing -std=c++17 will cause the build to fail. Do you know where is it handled in write_to_bazelrc?

Reply:
I test it it's not failing :)
Also, `tf_copts()` isn't applied to all targets, so that would be inconsistent to have c++17 only enabled for some targets!

But here is the source where it is handled in the build script:

https://github.com/tensorflow/lingvo/blob/master/pip_package/build.sh#L48-L50